engine/runtime/src/main/java/org/enso/interpreter/node/expression/literal/TextLiteralNode.java

```diff
@@ -34,4 +35,15 @@ public static TextLiteralNode build(String value) {
   public Text executeGeneric(VirtualFrame frame) {
     return value;
   }
+
+  @Override
+  public Object parsePatch(String text) {
+    if (text.length() >= 2 && text.charAt(0) == '"' && text.charAt(text.length() - 1) == '"') {
```

Comment:
This is indeed a brutal way of parsing a literal. The right way is: find the code that produces `IR.Literal.Text` (and `IR.Literal.Number`) and reuse exactly it. Any suggestions how to structure this communication between Scala and Java code without being too ugly, @kustosz , @hubertp ?